### PR TITLE
chore: handle relase automatically with github action

### PR DESCRIPTION
**Pull request by Slashgear**
- Handle release pipeline directly in github action
 - needs to be merged to be used
 - work on a new documentation of how to release when merged

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    inputs:
+      releaseParams:
+        description: 'Pass parameters to lerna publish (ex: major, minor, patch)'
+        required: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (Slashgear) [inline .github/workflows/release.yml:14]: @oltodo Is this the correct name of the secret ?

Comment (oltodo) [reply .github/workflows/release.yml:14]: It is.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          fetch-depth: 0
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: 'yarn'
+      - run: yarn install
+      - run: yarn lerna version --yes --conventional-commits --create-release github ${{ github.event.inputs.releaseParams }}
+      - run: yarn lerna publish from-git --yes
```
Comment (oltodo) [inline .github/workflows/release.yml:26]: Have you tested them?

Comment (Slashgear) [reply .github/workflows/release.yml:26]: That is what we use in bedrock js tools

Comment (Slashgear) [reply .github/workflows/release.yml:26]: I could try to make a pre-release from this branch locally with those commands in order to test 💡

**Review by oltodo — APPROVED** [review]

**Pull request merged**